applications/asset_tracker/src/main.c

```diff
@@ -1750,19 +1753,33 @@ void handle_bsdlib_init_ret(void)
 #endif /* CONFIG_BSD_LIBRARY */
 }
 
+static void no_sim_go_offline(struct k_work *work)
+{
+#if defined(CONFIG_BSD_LIBRARY)
+	lte_lc_offline();
+	k_sleep(K_MSEC(100));
+	LOG_INF("No SIM card detected.");
+	LOG_INF("Insert SIM and reset device to run the asset tracker.");
+	ui_led_set_pattern(UI_LED_ERROR_LTE_LC);
+#endif /* CONFIG_BSD_LIBRARY */
+}
+
 static void lte_handler(const struct lte_lc_evt *const evt)
 {
 #if defined(CONFIG_BSD_LIBRARY)
 	switch (evt->type) {
 	case LTE_LC_EVT_NW_REG_STATUS:
-		if ((evt->nw_reg_status != LTE_LC_NW_REG_REGISTERED_HOME) &&
-		     (evt->nw_reg_status != LTE_LC_NW_REG_REGISTERED_ROAMING)) {
-			break;
-		}
 
-		LOG_INF("Network registration status: %s",
+		if (evt->nw_reg_status == LTE_LC_NW_REG_UICC_FAIL) {
+			k_work_submit_to_queue(&application_work_q,
+				       &no_sim_go_offline_work);
+
+		} else if ((evt->nw_reg_status == LTE_LC_NW_REG_REGISTERED_HOME) ||
+		     (evt->nw_reg_status == LTE_LC_NW_REG_REGISTERED_ROAMING)) {
+			LOG_INF("Network registration status: %s",
```

Comment:
Is there something up with the indentation with the below lines, or just GitHub messing up the view?